README.md

```diff
@@ -1,62 +1,62 @@
 ES6: QuickStart and Tips
 ------------------------
 
-ES6: 快速体验，及实用小贴士
+ES6: quick experience, and practical tips
 
-> 2016 年是 ES6 大力推广和普及的黄金时期，也是今年的流行趋势，
-> 就一个 [ES6](https://github.com/search?o=desc&q=ES6&s=stars&type=Repositories&utf8=✓) 关键词，
-> 在 GitHub 上就有这么多搜索结果。（赶紧跟上大部队！）
+> 2016 is ES6 vigorously promote and popularize the golden age, but also this year's fashion trends,
+> On a [ES6] (https://github.com/search?o=desc&q=ES6&s=stars&type=Repositories&utf8=✓) keywords,
```

Comment:
Should `[ES6](https://github.com/search?o=desc&q=ES6&s=stars&type=Repositories&utf8=✓)`, no space